common.mk

```diff
@@ -156,4 +156,10 @@ $(output_dir)/tracegen.result: $(output_dir)/tracegen.out $(AXE)
 
 tracegen: $(output_dir)/tracegen.result
 
+dramsim_dir = $(base_dir)/tools/DRAMSim2
```

Comment:
Can we add some header comment to this section and move the `.PHONY` to be closer to the `tracegen` rule?

Something like this:
```
.PHONY: tracegen
tracegen: ...

#############################
# Target for DRAMSIM2 ....
#############################
```